mozdownload/__init__.py

```diff
@@ -2,4 +2,15 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from scraper import *
+from scraper import PLATFORM_FRAGMENTS, NotSupportedError, NotFoundError, \
+    NotImplementedError, TimeoutError, Scraper, DailyScraper, \
+    DirectScraper, ReleaseScraper, ReleaseCandidateScraper, \
+    TinderboxScraper, TryScraper
```

Comment:
I would suggest that we limit the number of exported symbols to what external code really needs. That means please remove `PLATFORM_FRAGMENTS` which is used only internally. For the others I would suggest to use one per line, and all surrounded by brackets, so you can avoid the backslashes. See the following link as an example: https://github.com/mozilla/firefox-ui-tests/blob/master/firefox_ui_harness/runtests.py#L13